spotmini:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  head_upperlegM1_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: base_upperlegM1_joint
    pid: {p: 10000, i: 0.01, d: 10.0}
  head_upperlegM2_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: base_upperlegM2_joint
    pid: {p: 10000, i: 0.01, d: 10.0}
  head_upperlegM3_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: base_upperlegM3_joint
    pid: {p: 10000, i: 0.01, d: 10.0}
  head_upperlegM4_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: base_upperlegM4_joint
    pid: {p: 10000, i: 0.01, d: 10.0}
  upperlegM1_lowerlegM1_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: upperlegM1_lowerlegM1_joint
    pid: {p: 10000, i: 0.01, d: 10.0}
  upperlegM2_lowerlegM2_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: upperlegM2_lowerlegM2_joint
    pid: {p: 10000, i: 0.01, d: 10.0}    
  upperlegM3_lowerlegM3_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: upperlegM3_lowerlegM3_joint
    pid: {p: 10000, i: 0.01, d: 10.0}
  upperlegM4_lowerlegM4_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: upperlegM4_lowerlegM4_joint
    pid: {p: 10000.0, i: 0.01, d: 10.0} 
  # joint9_position_controller:
  #   type: effort_controllers/JointPositionController
  #   joint: body1_joint
  #   pid: {p: 10000, i: 0.01, d: 10.0}
  # joint10_position_controller:
  #   type: effort_controllers/JointPositionController
  #   joint: body2_joint
  #   pid: {p: 10000, i: 0.01, d: 10.0}    
  # joint11_position_controller:
  #   type: effort_controllers/JointPositionController
  #   joint: body3_joint
  #   pid: {p: 10000, i: 0.01, d: 10.0}
  # joint12_position_controller:
  #   type: effort_controllers/JointPositionController
  #   joint: body4_joint
  #   pid: {p: 10000.0, i: 0.01, d: 10.0}  